src/components/Speciality/Projects.js

```diff
@@ -4,38 +4,58 @@ import { Row, Col, Grid } from '../grid'
 import { SectionTitle, CardTitle, Subtitle, BodyPrimary } from '../Typography'
 import { Padding } from 'styled-components-spacing'
 import breakpoint from 'styled-components-breakpoint'
+import getColorLuminance from '../../utils/getColorLuminance'
 import { AnimatedLink, CardHeader, PosterImage } from '../Common/animatedLink'
 import LogoGrid from '../Common/LogoGrid'
 
 const Emphasis = styled.em`
   color: ${props => props.theme.colors.secondaryText};
 `
 
-const PosterLinks = ({ project }) => (
-  <AnimatedLink to={`/case-study/${project.slug}`} title={project.title}>
-    <section
-      style={{
-        background: `#${project.posterColor}`
-      }}
-    >
-      <CardHeader>
-        <CardTitle reverse noPadding bigger>
-          {project.title}
-        </CardTitle>
-        <BodyPrimary reverse muted>
-          {project.introSentence.introSentence}
-        </BodyPrimary>
-      </CardHeader>
-      <PosterImage justifyCenter alignCenter color={project.posterColor}>
-        <img
-          alt={project.posterImage.title}
-          src={project.posterImage.file.url}
-          style={{ maxHeight: '100%' }}
-        />
-      </PosterImage>
-    </section>
-  </AnimatedLink>
-)
+const PosterLinks = ({ project }) => {
+  const imageKey =
+    project && project.alternativePreviewImage
+      ? 'alternativePreviewImage'
+      : 'posterImage'
+
+  const poster = {
+    title: project[imageKey].title,
+    url: project[imageKey].file.url,
+    color: project.posterColor
+  }
+
+  const isDarkPosterColor = getColorLuminance(poster.color) < 132.5
+
+  return (
+    <AnimatedLink to={`/case-study/${project.slug}`} title={project.title}>
+      <section
+        style={{
+          background: `#${poster.color}`
+        }}
+      >
+        <CardHeader>
+          <CardTitle reverse={isDarkPosterColor} noPadding bigger>
+            {project.title}
+          </CardTitle>
+          <BodyPrimary
+            muted
+            reverse={isDarkPosterColor}
+            lightMuted={isDarkPosterColor}
+          >
+            {project.introSentence.introSentence}
+          </BodyPrimary>
+        </CardHeader>
+        <PosterImage justifyCenter alignCenter color={poster.color}>
+          <img
+            alt={poster.title}
+            src={poster.url}
+            style={{ maxHeight: '100%' }}
+          />
```

Comment:
This should be the `Image` component from Common components. Same as [here](https://github.com/yldio/yld.io/blob/6fd316cf5d5747360b9c45d5599a5612189aa993/src/components/AboutUs/Subsidiaries.js#L14) but without the `width` music.

Instead of grabbing all the image data in `poster` and passing each of the keys down as prop values you can just do:

`<Image image={project[imageKey]} />`

This should do everything you need!